app/src/main/java/com/hedvig/app/util/featureflags/FeatureRuntimeBehavior.kt

```diff
@@ -0,0 +1,32 @@
+package com.hedvig.app.util.featureflags
+
+import androidx.annotation.VisibleForTesting
+import com.hedvig.app.feature.settings.MarketManager
+import java.util.concurrent.CopyOnWriteArrayList
+
+class FeatureRuntimeBehavior(
```

Comment:
This name still doesn't entirely make sense to me 😅 It appears to have the role of providing feature flags, I kind of wish it was named `FeatureFlagProvider` 😬

Reply:
Agreed, changing :)